articles/openshift/oshift-how-backup-directories-database.md

```diff
@@ -0,0 +1,207 @@
+---
+title: How to backup container directories, databases and restore them on OpenShift | UKCloud Ltd
+description: Provides guidance on how customers can backup container directories, databases and restore them. 
+services: openshift
+author: Daniel Brennand
+reviewer: 
+lastreviewed: 
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Backup container directories, databases and restore them.
+toc_fullpath: How To/oshift-how-backup-directories-database.md
+toc_mdlink: oshift-how-backup-directories-database.md
+---
+
+# How to backup container directories, databases and restore them
+
+## Overview
+
+This article will explain how you can backup container directories, databases (specifically [PostgreSQL](https://www.postgresql.org/)) to your localhost. Furthermore, it will also go through how you can restore them.
```

Comment:
Using "localhost" here feels like an odd turn of phrase.
```suggestion
This article will explain how you can backup container directories, databases (specifically [PostgreSQL](https://www.postgresql.org/)) to your local client. Furthermore, it will also go through how you can restore them.
```

Reply:
I prefer local client as well, if that counts for anything ;)